lib/Cleantalk/Custom/Antispam/lock-page-ct-die.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
```

Comment:
Why this class is in `Custom` namespace? Is not it really enough the common class?

Reply:
relative path